documentation/ame-guide/modules/ROOT/pages/edit-elements.adoc

```diff
@@ -328,6 +328,20 @@ NOTE: Before you can add a Constraint to your Aspect Model, you have to add a Tr
 
 For more information on adding Traits, see xref:edit-models.adoc#trait-case[Use Traits].
 
+[[custom-units]]
+== Custom Units
+
+The Aspect Meta Model includes a comprehensive catalog of physical units and quantity kinds, providing a rich foundation for representing various quantities. For instance, the catalog encompasses predefined units like "metre," "inch," "millimetre," and more, under the quantity kind of "length." This catalog aligns with the UNECE Recommendation 20 [rec20] and offers approximately 1,800 predefined units.
```

Comment:
For Rec20, either the text "UNECE Recommendation 20" should be a link to the Rec20 website, or the "[rec20]" should be a link to the SAMM spec's appendix, i.e., either `xref:samm-specification:appendix:bibliography.adoc#rec20[[rec20\]]` or `https://eclipse-esmf.github.io/samm-specification/snapshot/appendix/bibliography.html#rec20`